### PR TITLE
Add version number to UI

### PR DESCRIPTION
**Pull request by juanjoDiaz**
A second proposal for #1222.
An alternative to #1264.

### CODE REVIEW

File: vnc.html [hunk 1]
```diff
@@ -259,6 +259,11 @@ <h1 class="noVNC_logo" translate="no"><span>no</span><br>VNC</h1>
                             </li>
                         </ul></div>
                     </li>
+                    <li><hr></li>
+                    <li>
+                        Version:
+                        <span class="noVNC_version"></span>
```
[the remark this answers is not in the document]
Comment (juanjoDiaz) [reply vnc.html]: What do you mean?

[the remark this answers is not in the document]
Comment (samhed) [reply vnc.html]: Oh, sorry. Didn't realize html tags didn't show on github. I tried to write:

> Could you wrap all of this in `<small>`?

I was thinking the version number could be a little smaller than the rest of the text in the settings.

[the remark this answers is not in the document]
Comment (CendioOssman) [reply vnc.html]: That is better handled in CSS, so give it a proper class and set up some rules there.

File: app/ui.js [hunk 2]
```diff
@@ -59,6 +59,11 @@ const UI = {
         // Translate the DOM
         l10n.translateDOM();
 
+        WebUtil.fetchJSON('../package.json')
```
Comment (CendioOssman) [inline app/ui.js]: What happens if this fails? It's not terribly obvious that `package.json` is needed for the web pages, so people might be deploying without it.

Comment (juanjoDiaz) [reply app/ui.js]: The idea solution imho would be to package noVNC before deployment (with rollup or such).
That way, the number just get injected here and there is no need to import the whole package.json file just for the number.
But you want the git repo to be deployable as is, so this is the only possibility I came up with.

I can add a catch case and simply say `Unkown` if package.json is not there.

**Comment by juanjoDiaz** [comment]
Should be better now.

The font is smaller.
A whitespace is added before the number.
If fetching package.json fails, the whole version entry is removed from the menu.

File: vnc.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (samhed) [reply vnc.html]: True

File: app/ui.js (hunk 2 above)
Comment (samhed) [reply app/ui.js]: > I can add a catch case and simply say Unkown if package.json is not there.

Good enough for me

**Pull request merged**